.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "source/dependency/coreJSON"]
+	path = source/dependency/coreJSON
+	url = https://github.com/FreeRTOS/coreJSON
+	update = none
```

Comment:
Curious why not lock in a specific commit?

Reply:
In git submodule, the commit of this repo is bound automactically. Take [this folder](https://github.com/ActoryOu/amazon-kinesis-video-streams-signaling-public/tree/aggarg_suggestions/source/dependency) as reference. It shows that [dc1ab91](https://github.com/FreeRTOS/coreJSON/tree/dc1ab9130a1fb99b801a2a1fa8e9f42239f752be) is the commit bound in signaling repo.

Reply:
Makes sense, thanks for the clarification!